src/lib/consensus/consensus.ml

```diff
@@ -5,11 +5,6 @@ module Constants = Constants
 include Intf
 
 [%%if
```

Comment:
Delete?

Reply:
I left this to still provide an error if there is a configuration with a `consensus_mechanism` value other than `"proof_of_stake"` (note the `[%%error ...]` in the else case).